src/components/grid-aware/StatsBarBlock/StatsBarBlock.module.css

```diff
@@ -0,0 +1,218 @@
+.gray {
+  background-color: var(--color-gray-300);
+}
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+.bleedBackground {
+  grid-column: 2 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0 150px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 38px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column: 2 / span 10;
+  grid-row: 3;
+  margin-top: 54px;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  grid-column: 2 / span 8;
+  margin: 0;
+}
+
+.blue {
+  color: var(--color-sheltertech-blue);
+}
+
+.subtitle {
+  font: var(--font-body-medium);
+  grid-column: 2 / span 8;
+  margin-top: 20px;
+}
+
+.card {
+  flex: 1 1 0;
+  margin-right: 110px;
+  width: 250px;
+}
+
+.card:last-child {
+  margin-right: 0;
+}
+
+@media (--tablet-and-down) {
+  .card {
+    margin-bottom: 40px;
+  }
+
+  .card:first-child {
+    margin-top: 40px;
+  }
+}
+
+.cardSubtitle {
+  color: var(--color-gray-700);
+  font: var(--font-body-small);
+  margin-top: 5px;
+}
+
+.cardTitle {
+  font: var(--font-body-medium);
+  margin-top: 5px;
+}
+
+.barsContainer {
+  display: grid;
+}
+
+.barsContainer.vertical {
+  column-gap: 26px;
```

Comment:
<img width="166" alt="Screen Shot 2022-03-16 at 11 17 53 PM" src="https://user-images.githubusercontent.com/1002748/158748648-7eba76bd-9ecc-4783-8b1c-a2d612f42d74.png">

It's subtle, but I think there's actually a column gap of 20px here. The trick is that the year label at the bottom is actually the widest part of the column, not the bar nor the stat number, and the spacing between the labels is 20px.

I also noticed that when I tweaked this number to 20px, it becomes a lot more obvious that the bars are not properly center-aligned to the year label. You'll probably also need to change the alignment (or justification, because I always mix them up) to make sure the bars are centered over the labels.